config/capsule-answers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+certs:
+  generate: false
+  deploy: true
+capsule: true
```

Comment:
another option is to set new default in `puppet-capsule`. If there are reasons against i'd go with the 
migration.